drivers/go-getting-started.md

```diff
@@ -15,6 +15,58 @@ description: To use the driver, first fetch the sources into your GOPATH
 
 - None (Additional error libraries are supported).
 
+## Usage
+
+To use the Go driver, import it as a Go package into your program
+using the `import` statement, then open a client connection to a URL,
+with username and password, using the API. This results in a handle
+with which to create and edit objects for collections, documents,
+nodes, and edges. These database objects are mapped to types in
+Go. The methods for these types are used to read and write data.
+
+Some operations (like deleting a database) cannot easily be done using
+the API, as they are too dangerous.
+
+## Go idioms
+
+Go uses package names as datatypes, and datatypes as qualifiers, to invoke 
+member functions:
+
+```go
+package.member()
+var myvariable package.typename 
```

Comment:
This is unexported type. You can't use unexported types in go. Please convert to `package.TypeName`